api/api/tests/test_serializers.py

```diff
@@ -122,3 +122,34 @@ def test_jaar_ongeval_quickscan_missing_none(self):
             self.serializer.validate(attrs)
             self.assertIn('jaar_ongeval_quickscan', attrs)
             self.assertEqual(attrs['jaar_ongeval_quickscan'], None)
+
+    def test_validate_spot_types_missing_jaarblackspotlijst(self):
```

Comment:
In this test and the one below I would add a comment what kind of ValidationError you're expecting and why. This will help future readers.

Reply:
Done, zie https://github.com/Amsterdam/blackspots-backend/pull/42/commits/63098bcfcc390c56a69be0e175e79d7ab779d8eb